### DISCUSSION TITLE
Change default dashboard password at cluster creation

### DISCUSSION THREAD

**Discussion by TxMat** [emqx/emqx#13161] [Q&A] [answered]
Hi,

We are creating and destroying multiple emqx cluster a day in our infra without any persistance and we are currently using a predefined cluster.hocon as well as `__` env variables to configure everything but i didn't found any way to change the dashboard password before the cluster creation. Is this possible ? does this relate to #12972

Thanks !

**Comment by id** [comment, marked as answer]
Yeah, use `EMQX_DASHBOARD__DEFAULT_PASSWORD`.
